src/bomsquad/vulndb/client/nvd.py

```diff
@@ -71,10 +74,10 @@ def products(
         while True:
             url = f"{self.CPE_STEM}?startIndex={offset}"
             if last_mod_start_date:
-                url += f"&lastModStartDate={last_mod_start_date.isoformat()}&lastModEndDate={datetime.now().isoformat()}"
-                logger.info(
-                    f"Querying from {offset} - {limit} and {last_mod_start_date.isoformat()} - {datetime.now().isoformat()}"
-                )
+                dtstart = (last_mod_start_date + timedelta(microseconds=1000)).isoformat()
```

Comment:
For timezone-aware datetime in utc, it would be a little more direct and clear to datetime.now(timezone.utc) rather than utcnow with replace.